### ISSUE TITLE
test_unicode_characters_tar failing in 0.11.0rc3

### ISSUE THREAD

**Issue by katre** [bazelbuild/bazel#4626]
https://ci.bazel.build/blue/organizations/jenkins/Global%2Fbazel-tests/detail/bazel-tests/500/tests

Test is failing on the platforms:
- linux-x86_64,python_version=2
- linux-x86_64,python_version=3
- ubuntu_16.04-x86_64

Error:
```
ext/
ext/unicode-\360\237\215\203.txt
ext/hello.txt
ext/BUILD
INFO: $TEST_TMPDIR defined: output root default is '/home/ci/.cache/bazel/_bazel_ci/65ed5a6a0442d902ec235fee508231f8/bazel-sandbox/5230614367809572249/execroot/io_bazel/_tmp/641f91430ea2b499e1c7c88c29a221a9' and max_idle_secs default is '15'.
........................
Loading:
Loading: 0 packages loaded
ERROR: Skipping '@ext//:hello.txt': no such package '@ext//': Malformed input or input contains unmappable characters: /home/ci/.cache/bazel/_bazel_ci/65ed5a6a0442d902ec235fee508231f8/bazel-sandbox/5230614367809572249/execroot/io_bazel/_tmp/641f91430ea2b499e1c7c88c29a221a9/root/ee117947db37c640b7a7f72a55cbc0d0/external/ext/unicode-🍃.txt
WARNING: Target pattern parsing failed.
ERROR: no such package '@ext//': Malformed input or input contains unmappable characters: /home/ci/.cache/bazel/_bazel_ci/65ed5a6a0442d902ec235fee508231f8/bazel-sandbox/5230614367809572249/execroot/io_bazel/_tmp/641f91430ea2b499e1c7c88c29a221a9/root/ee117947db37c640b7a7f72a55cbc0d0/external/ext/unicode-🍃.txt
INFO: Elapsed time: 1.804s
FAILED: Build did NOT complete successfully (0 packages loaded)
-- Test log: -----------------------------------------------------------
INFO: $TEST_TMPDIR defined: output root default is '/home/ci/.cache/bazel/_bazel_ci/65ed5a6a0442d902ec235fee508231f8/bazel-sandbox/5230614367809572249/execroot/io_bazel/_tmp/641f91430ea2b499e1c7c88c29a221a9' and max_idle_secs default is '15'.
...............................
INFO: Starting clean (this may take a while). Consider using --async if the clean takes more than several minutes.
------------------------------------------------------------------------
test_unicode_characters_tar FAILED: expected success .
/home/ci/.cache/bazel/_bazel_ci/65ed5a6a0442d902ec235fee508231f8/bazel-sandbox/5230614367809572249/execroot/io_bazel/bazel-out/k8-fastbuild/bin/src/test/shell/bazel/external_integration_test.runfiles/io_bazel/src/test/shell/bazel/external_integration_test:976: in call to test_unicode_characters_tar
```

**Comment by katre** [comment]
Blocking release https://github.com/bazelbuild/bazel/issues/3959

**Comment by aehlig** [comment]
On Mon, Feb 12, 2018 at 09:47:17AM -0800, katre wrote:
> https://ci.bazel.build/blue/organizations/jenkins/Global%2Fbazel-tests/detail/bazel-tests/500/tests
> 
> Test is failing on the platforms:
> - linux-x86_64,python_version=2
> - linux-x86_64,python_version=3
> - ubuntu_16.04-x86_64

FYI, this test was rolled back in 109e4b4dc9e786e3a2d8d7cb245d18320dbe9216

-- 
Klaus Aehlig
Google Germany GmbH, Erika-Mann-Str. 33, 80636 Muenchen
Registergericht und -nummer: Hamburg, HRB 86891
Sitz der Gesellschaft: Hamburg
Geschaeftsfuehrer: Paul Terence Manicle, Halimah DeLaine Prado

**Comment by katre** [comment]
Thanks. I will cherrypick that to the next RC.

**Comment by katre** [comment]
Fixed in 0.11.0rc4.
